product_docs/docs/pgd/5.6/commit-scopes/synchronous_commit.mdx

```diff
@@ -40,4 +39,5 @@ SELECT bdr.add_commit_scope(
 
 ## Details
 
-Currently `SYNCHRONOUS_COMMIT` doesn't use the confirmation levels of the commit scope rule syntax.
+Currently `SYNCHRONOUS COMMIT` doesn't use the confirmation levels of the commit scope rule syntax. `SYNCHRONOUS COMMIT` in commit scope rules has an alias, `SYNCHRONOUS_COMMIT`. The use of the space instead of the underscore helps distinguish it from Postgres's native `SYNCHRONOUS_COMMIT`.
```

Comment:
```suggestion
Currently `SYNCHRONOUS COMMIT` doesn't use the confirmation levels of the commit scope rule syntax.

In commit scope rules, the original keyword `SYNCHRONOUS_COMMIT` is now aliased to `SYNCHRONOUS COMMIT`. The use of a space instead of an underscore helps distinguish it from Postgres's native `SYNCHRONOUS_COMMIT`.
```